File: python/GreatestOfThree.py
a=input("Enter first number\n")
b=input("Enter second number\n") 
c=input("Enter third number\n")
if  int(a)>int(b) and int(a)>int(c):
    print(a + " is greatest")
elif int(b)>int(a) and int(b)>int(c):
    print(b + " is greatest")
elif int(c)>int(a) and int(c)>int(b):
    print(c + " is greatest")
else:
    print("all number are equal") 

File: dart/GreatestOfThree.dart
import 'dart:io';

int main() {
  print("Enter three numbers: ");
  int? a = int.parse(stdin.readLineSync()!);
  int? b = int.parse(stdin.readLineSync()!);
  int? c = int.parse(stdin.readLineSync()!);
  if (a > b && a > c) {
    print('$a' "is the greatest number");
  } else if (b > a && b > c) {
    print('$b' " is the greatest number");
  } else if (a > b && a > c) {
    print('$c' ' is the greatest number');
  } else
    () {
      print("all numbers are same");
    };
  return 0;
}
